editor/js/Strings.js

```diff
@@ -133,6 +133,7 @@ function Strings( config ) {
 			'sidebar/geometry/bounds': 'Bounds',
 			'sidebar/geometry/show_vertex_normals': 'Show Vertex Normals',
 			'sidebar/geometry/compute_vertex_normals': 'Compute Vertex Normals',
+			'sidebar/geometry/compute_vertex_tangents': 'Compute Tangents',
```

Comment:
@mrdoob I've used the shorter name `Compute Tangents` since `Compute Vertex Tangents` produces a line break inside the button.